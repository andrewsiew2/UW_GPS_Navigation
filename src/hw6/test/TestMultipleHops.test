# Tests for if a path from a node to another with a multiple hops can be found

LoadGraph tester testing.tsv

FindPath tester WONDER_WOMAN THOR