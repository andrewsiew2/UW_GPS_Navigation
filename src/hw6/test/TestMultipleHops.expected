# Tests for if a path from a node to another with a multiple hops can be found

loaded graph tester

path from WONDER WOMAN to THOR:
WONDER WOMAN to HULK via 25
HULK to ROBIN via 90
ROBIN to THOR via 23